pkg/controller/s2irun/s2irun_controller.go

```diff
@@ -220,6 +220,13 @@ func (r *ReconcileS2iRun) Reconcile(request reconcile.Request) (reconcile.Result
 		if found.Status.Active == 1 {
 			log.Info("Job is running", "start time", found.Status.StartTime)
 			instance.Status.RunState = devopsv1alpha1.Running
+			if instance.Status.LogURL == "" {
```

Comment:
remove this line is ok too because in level-based api, we don't care what was it like but what it should be now